lib/mapObjects/CGHeroInstance.cpp

```diff
@@ -935,18 +935,14 @@ si32 CGHeroInstance::getManaNewTurn() const
 // 	ai->putAt(this, ai->firstAvailableSlot(this));
 // }
 
-int CGHeroInstance::getBoatType() const
+TransportId CGHeroInstance::getTransportType() const
 {
 	switch(type->heroClass->getAlignment())
 	{
-	case EAlignment::GOOD:
-		return 1;
-	case EAlignment::EVIL:
-		return 0;
-	case EAlignment::NEUTRAL:
-		return 2;
-	default:
-		throw std::runtime_error("Wrong alignment!");
+		case EAlignment::EVIL : return TransportId::ETransportId::BOAT_EVIL;
```

Comment:
Move to hero class config?

Reply:
out of this PR

Reply:
no, didn't plan to refactor hero unless there will be such request